submitted_models/marble_spot_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,2172 @@
+                <dynamic_bias_correlation_time>175</dynamic_bias_correlation_time>
+                <precision>0.005</precision>
+              </noise>
+            </y>
+            <z>
+              <noise type="gaussian">
+                <mean>0</mean>
+                <stddev>0.021</stddev>
+                <bias_mean>0.05</bias_mean>
+                <bias_stddev>0.0075</bias_stddev>
+                <dynamic_bias_stddev>0.000375</dynamic_bias_stddev>
+                <dynamic_bias_correlation_time>175</dynamic_bias_correlation_time>
+                <precision>0.005</precision>
+              </noise>
+            </z>
+          </linear_acceleration>
+        </imu>
+      </sensor>
+    </link>
+    <joint name="laser_j" type="fixed">
+      <pose relative_to="payload">0.305 0 0.2325 0 -0 0</pose>
+      <parent>payload</parent>
+      <child>laser</child>
+      <axis>
+        <dynamics>
+          <spring_reference>0</spring_reference>
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>-1e+16</lower>
+          <upper>1e+16</upper>
+        </limit>
+      </axis>
+      <physics>
+        <ode>
+          <limit>
+            <cfm>0</cfm>
+            <erp>0.2</erp>
+          </limit>
+        </ode>
+      </physics>
+    </joint>
+    <link name="laser">
+      <pose relative_to="laser_j">0 0 0 0 -0 0</pose>
+      <inertial>
+        <pose>0 0 0.03675 0 -0 0</pose>
+        <mass>0.5</mass>
+        <inertia>
+          <ixx>0.000461625</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.000461625</iyy>
+          <iyz>0</iyz>
+          <izz>0.000473063</izz>
+        </inertia>
+      </inertial>
+      <collision name="laser_collision">
+        <pose>0 0 0.03675 0 -0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.0735</length>
+            <radius>0.0435</radius>
+          </cylinder>
+        </geometry>
+      </collision>
+      <visual name="laser_visual">
+        <geometry>
+          <mesh>
+            <uri>meshes/ouster-os0.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <sensor name="laser" type="gpu_lidar">
+        <pose>0 0 0.035925 0 -0 0</pose>
+        <update_rate>10</update_rate>
+        <visualize>0</visualize>
+        <always_on>1</always_on>
+        <lidar>
+          <scan>
+            <horizontal>
+              <samples>1024</samples>
+              <resolution>1</resolution>
+              <min_angle>-3.1459</min_angle>
+              <max_angle>3.1459</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>64</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.2897</min_angle>
+              <max_angle>0.2897</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.1</min>
+            <max>50</max>
```

Comment:
`specification.md` states that this lidar has a range of 120m. Is there a reason for the value of 50 here?

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d